dts/bindings/ufs/ufs-common.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2025 Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Common properties for Universal Flash Storage (UFS) Host Controllers
+
+include: [base.yaml]
+
+properties:
+  freq-table-hz:
+    type: array
+    description: |
+      Array of <min max> operating frequencies in Hz stored in the same order
+      as the clocks property. If this property is not defined or a value in the
+      array is "0" then it is assumed that the frequency is set by the parent
+      clock or a fixed rate clock source.
+
+  interrupts:
+    required: true
+    description: Interrupts used by the UFS Host Controller.
+
+  lanes-per-direction:
+    type: int
+    enum: [1, 2]
+    default: 2
+    description:
+      Number of lanes available per direction. Note that it is assume same
+      number of lanes is used both directions at once.
```

Comment:
Please extend the description to account for the default following the rules in https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values -- if you can't make up a "why" for this default that follows the rules, please make this required